trillian/integration/ct_functions.sh

```diff
@@ -37,7 +37,7 @@ ct_prep_test() {
     "${CT_GO_PATH}/scripts/resetctdb.sh" --force
     # Wipe the incident database
     echo "Wiping and re-creating incident database"
-    "${MONO_PATH}/scripts/resetmondb.sh" --force
+    bash "${MONO_PATH}/scripts/resetmondb.sh" --force
```

Comment:
Probably should fix this upstream too (and then later remove the workaround)?

Reply:
By upstream do you mean https://github.com/golang/go/issues/27455 ?
I've already fixed it in the Trillian repo here: https://github.com/google/trillian/pull/1673

Reply:
Oh, it's "WAI" for go mod? Readonly seems sensible, but !x seems sad. Oh well.

Reply:
I don't think it's WAI, based on https://github.com/golang/go/issues/31481#issuecomment-500415155. However, until they fix it, we have to workaround it.